python/lsst/daf/butler/registries/oracleRegistry.py

```diff
@@ -67,10 +70,20 @@ def setConfigRoot(cls, root, config, full):
     def __init__(self, registryConfig, schemaConfig, dimensionConfig, create=False,
                  butlerRoot=None):
         registryConfig = SqlRegistryConfig(registryConfig)
+
+        # currently code in class specific to cx_oracle driver
+        url = engine.url.make_url(registryConfig[".db.url"])
```

Comment:
The preference is to write the config access as : `["db", "url"]` rather than using a delimiter.